Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -115,7 +116,8 @@ vtkMRMLSliceNode::vtkMRMLSliceNode()
   this->LayoutColor[1] = vtkMRMLSliceNode::grayColor()[1];
   this->LayoutColor[2] = vtkMRMLSliceNode::grayColor()[2];
 
-  this->SetOrientationToAxial();
+  this->SetOrientationString("Reformat");
+  this->SetOrientationReference("Reformat");
```

Comment:
in priciple yes, however SetOrientationReference and SetOrientationString are vtkSetMacro. Should I modify them to "normal" methods? or is it possible in some way to call the vtkSetMacro inside another vtkSetMacro?